readme.md

```diff
@@ -164,7 +164,7 @@ Join us on [GH Discussions][chat]!
 
 ## License
 
-[MIT][] © [Compositor][] and [Vercel][]
+[MIT][] © [Components AI][] and [Vercel][]
```

Comment:
did the license transfer? :thinking: 
/cc @johno @wooorm

Reply:
Copyright doesn’t transfer like this. Compositor !== Components AI as far as I can tell, so this change is incorrect.
We can drop the link to `Compositor` though, if it’s no longer alive

Reply:
OK, I'll remove it then.

Reply:
Done!